sap/cf_logging/falcon_logging/__init__.py

```diff
@@ -0,0 +1,62 @@
+import logging
+from datetime import datetime
+
+import falcon
+
+from sap import cf_logging
+from sap.cf_logging import defaults
+from sap.cf_logging.core.constants import REQUEST_KEY, RESPONSE_KEY
+from sap.cf_logging.core.framework import Framework
+from sap.cf_logging.falcon_logging.context import FalconContext
+from sap.cf_logging.falcon_logging.request_reader import FalconRequestReader
+from sap.cf_logging.falcon_logging.response_reader import FalconResponseReader
+
+FALCON_FRAMEWORK_NAME = 'falcon.framework'
+
+
+class LoggingMiddleware:
+
+    def __init__(self, logger='cf.falcon.logger'):
+        self.logger = logger
```

Comment:
The name `logger` suggest an object, but this is a name, can you name it like `logger_name` for better readability?